features/dev/verblijfplaats-binnenland-verblijfadres.feature

```diff
@@ -21,10 +21,10 @@ Functionaliteit: Persoon: verblijfplaats binnenland - verblijfadres
     | type | Adres  |
 
     Voorbeelden:
-    | gba naam           | gba waarde | naam                     |
-    | straatnaam (11.10) | .          | verblijfadres.straat     |
-    | huisnummer (11.20) | 0          | verblijfadres.huisnummer |
-    | woonplaats (11.70) | .          | verblijfadres.woonplaats |
+    | gba naam           | gba waarde | naam                              |
```

Comment:
deze scenario kan worden verwijderd. Het staat al in features/bevragen/persoon/verblijfplaats/adres/verblijfadres/onbekendwaardes.feature

Reply:
Verwijderd